Lib/ufo2ft/featureWriters/ast.py

```diff
@@ -51,16 +51,15 @@ def findFeatureTags(feaFile):
 
 def findCommentPattern(feaFile, pattern):
     """
-    Return the parent and the comment matching a given pattern.
-    Parent is either None or a block.
+    Return a tuple with the block (or nested blocks) and the comment matching a given pattern.
```

Comment:
technically it doesn't "return" since it's a generator, it "yields" a tuple of statements, where the last one is a comment matching the patter and the previous ones are the comment's (nested) parent blocks

Reply:
I replaced it with this:
```
   """
    Yield a tuple of statements, starting with the parent block, followed by
    nested blocks if present, ending with the comment matching a given pattern.
    There is not parent block if the matched comment is a the root level.
   """
```